buildroot-external/patches/occu/0093-WebUI-AllowAdminUserChanges.patch

```diff
@@ -0,0 +1,41 @@
+--- occu/WebUI/www/rega/esp/system.fn.orig
++++ occu/WebUI/www/rega/esp/system.fn
+@@ -1164,7 +1164,7 @@
+     oUser = dom.GetObject( system.GetVar("userid") );
+     if( oUser )
+     {
+-      if( !oUser.Unerasable() )
++      if( !oUser.Unerasable() || oUser.ID() == 1004 )
```

Comment:
The default user 'Admin' has the attribute `Unerasable` set.
I did not find a way to set or revoke this attribute within the WebUI.
So it seems that 'Admin' is system wide the only user with this attribute.

But I don't want to remove the check for `Unerasable`, therefore I added the user id `1004` to be allowed getting deleted, even if the `Unerasable` attribute is set.

Reply:
Did you test that after having deleted "Admin" everything still works fine. And if you create a new "Admin" user or admin user with a different name will it be set `Unerasable` as well?

Reply:
At the moment I can't do any further tests, because VirtualBox is not working on macOS BigSur and I have to wait for a fix :/

Reply:
Good to know :) So I will postpone my update to Big Sur also because macports is not ready yet.

Ok, then I will do these tests myself. Thx anyway.

Reply:
@jens-maus Regarding VirtualBox: It is now running under Bug Sur
This workaround does the trick for me:
https://forums.virtualbox.org/viewtopic.php?f=39&t=98763&start=75#p488559